.github/workflows/ci.yml

```diff
@@ -17,17 +17,28 @@ jobs:
       - name: Install uv
         uses: astral-sh/setup-uv@v4
         with:
-          version: "0.5.7"
+          version: "0.6.2"
           python-version: ${{ matrix.python-version }}
 
-      - name: Install dependencies
-        run: uv sync --all-extras
+      - name: Install "all" dependencies
+        run: uv sync --extra all
```

Comment:
since nothing is extra anymore... shall we change this to the mlip or something like that.